internal/pkg/docker/dockerfile/dockerfile.go

```diff
@@ -249,10 +250,11 @@ func parseHealthCheck(content string) (*HealthCheck, error) {
 		return nil, err
 	}
 
-	// cmdIndex holds the starting and ending index of "CMD"
+	// cmdIndex holds the starting index of "CMD", -1 if not present
```

Comment:
This comment is not needed since it can't be -1

Reply:
Ah good catch, I forgot to take that out while testing